about/press_kit/index.markdown

```diff
@@ -21,54 +23,91 @@ Specific version references of MoveIt framework:
 - All specific mentions of the MoveIt are written as "MoveIt 2", "MoveIt 3", etc.
 - All mentions of the MoveIt 2 port will be written as “MoveIt 2” or as “the MoveIt port to ROS 2.0"
 
-## Logo Guidelines
 
-We ask that you not:
+## Trademark Use ##
 
-- Place a logo in such close proximity to other content that it is indistinguishable
-- Make the logo the most distinctive or prominent feature on your website, printed material or other content
-- Use the logos in a way that is harmful, deceptive, obscene or otherwise objectionable to the average person
-- Use the logos to, or in connection with, content that disparages us or sullies MoveIt’s reputation
+Subject to compliance with this Policy, you may use the "MoveIt" name or the MoveIt logo to indicate your implementation of the MoveIt platform as set forth in this Policy. All other uses of the MoveIt trademarks must be expressly approved in writing by PickNik Robotics. To inquire about approval for any such other uses of the MoveIt trademarks, please contact hello@picknik.ai.
 
-The logo files are available under the [Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/) license.
+Any and all use of the "MoveIt" name, the MoveIt logo, and other MoveIt trademarks inures to the benefit of PickNik Robotics. You agree not to do anything inconsistent with PickNik Robotics' ownership of these trademarks, including without limitation seeking registration of any mark, domain name, or social media name/handle comprised of or incorporating the "MoveIt" name, the MoveIt logo, or any other MoveIt trademarks. You likewise agree not to make any use of the MoveIt trademarks that is likely to confuse or mislead consumers with respect to any relationship with PickNik Robotics or to suggest that PickNik Robotics sponsors, endorses, or approves your product or implementation; and you agree not to make any use of the MoveIt trademarks that is likely to harm the reputation and goodwill built up in the marks. You acknowledge and agree that any and all permissions to use any MoveIt trademarks are limited, personal, non-transferable, non-sublicensable, and revocable. PickNik Robotics may revoke any such permissions in its sole and absolute discretion.
+
+
+## Trademark Style Guidelines ##
 
-## Download
 
-Note for any print media it is necessary to use the provided .AI file and print in the provided CYMK values.
+#### MoveIt in text ####
 
-- Digital Color 0A58F7
-- Print Color (CMYK) C96 M64 Y0 K3
 
-### MoveIt
+*   MoveIt™ should have a trademark symbol the first time it appears in a creative.
+*   MoveIt should always be presented in camel case and is never plural.
```

Comment:
TLDR;
*  camelCase
*  PascalCase
*  snake_case
*  kebab-case

It is MoveIt not moveIt, so it is `pascal case`.

Reply:
https://en.wikipedia.org/wiki/Camel_case

You could also call it `upper camel case` but `pascal case` is clearer and imo the more common term for this type of capitalization scheme.

Reply:
I think UpperCamelCase is a lot clearer. It needs no explanation if you know camelCase, but without googling you will never guess what the connection of PascalCase to the SI unit of pressure, or the French philosopher, or the programming language might be.